name.abuchen.portfolio/src/name/abuchen/portfolio/money/LimitPrice.java

```diff
@@ -0,0 +1,80 @@
+package name.abuchen.portfolio.money;
+
+import java.util.Objects;
+
+public class LimitPrice implements Comparable<LimitPrice>
```

Comment:
Zu dem LimitPrice
- Bitte in das "model" Java Package auf der Ebene wie SecurityPrice
- Es wäre super wenn Du in der ```ClientFactory``` für stream noch einen Kurznamen (z.B. "limitPrice") hinterlegen könntest. Ansonsten wird das XML mit den Packagenamen unschön (und groß).
- Die Versionsnummer von Client und in ClientFactory (die ```ClientFactory#upgradeModel``` Method) müssen noch hochgezählt werden weil alte PP Versionen diese Datei nicht mehr laden können
- Ich vermeide es nach Möglichkeit das eine Klasse LimitPrice ein gleichnamiges Attribut limitPrice hat. Vielleicht doch ```value```?


```xml
            <name.abuchen.portfolio.money.LimitPrice>
              <compareType>GREATER_OR_EQUAL</compareType>
              <limitPrice>200000</limitPrice>
            </name.abuchen.portfolio.money.LimitPrice>
```

Reply:
Hab ich so übernommen. Wird nun korrekt im XML abgespeichert. Die Klasse `LimitPrice` hat nun einen `compareType` und einen `value`